include/sounds.h

```diff
@@ -555,6 +555,7 @@ enum SoundBank {
 #define SOUND_MENU_HIGH_SCORE                    /* 0x70222081 */ SOUND_ARG_LOAD(SOUND_BANK_MENU,     0x22, 0x20, SOUND_DISCRETE) // unverified
 #define SOUND_MENU_STAR_SOUND_OKEY_DOKEY         /* 0x7023FF81 */ SOUND_ARG_LOAD(SOUND_BANK_MENU,     0x23, 0xFF, SOUND_DISCRETE)
 #define SOUND_MENU_STAR_SOUND_LETS_A_GO          /* 0x7024FF81 */ SOUND_ARG_LOAD(SOUND_BANK_MENU,     0x24, 0xFF, SOUND_DISCRETE)
+#define SOUND_MENU_CRITICAL_ERROR                                 SOUND_ARG_LOAD(SOUND_BANK_MENU,     0x40, 0xFF, SOUND_DISCRETE)
```

Comment:
Add a comment with the hex for this, to be consistent with the other sounds.

Reply:
The hex in general within that file should be removed to be consistent with the repo. Those addresses don’t even line up with anything since they’re from decomp.

Reply:
Nevermind I realize it’s just a magic number not an address lmao